app/src/main/res/layout/activity_main.xml

```diff
@@ -0,0 +1,131 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout>
+
+    <androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:app="http://schemas.android.com/apk/res-auto"
+        xmlns:tools="http://schemas.android.com/tools"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:background="@color/black40"
+        tools:context=".MainActivity">
+
+        <TextView
+            android:id="@+id/textview_info_heading"
+            android:layout_width="211dp"
+            android:layout_height="20dp"
+            android:layout_marginStart="82dp"
+            android:layout_marginTop="86dp"
+            android:layout_marginEnd="82dp"
+            android:text="Thông tin sản phẩm nhập kho"
+            android:textFontWeight="400"
```

Comment:
Thay vì làm như này, nếu muốn viết chữ in đậm trong Android anh có thể sử dụng thuộc tính `textStyle`